features/admins/admin_post_issues.feature

```diff
@@ -14,6 +14,10 @@ Feature: Admin Actions to Post Known Issues
       And I press "Post"
     Then I should see "Known issue was successfully created"
       And I should see "First known problem"
+    When I follow "Admin Posts"
+      And I follow "Known Issues" within "#header"
+    Then I should see "Known Issues"
```

Comment:
It would be a good idea to specify a selector here (something like `#main ul.navigation`, but you'll have to double check that it's specific enough), since the text we're looking for appears in multiple places on the page. If our "Known Issues" navigation link wasn't appearing, this test might still pass because the words "Known Issues" also appear in the admin header navigation, the page's h2, and the site footer.